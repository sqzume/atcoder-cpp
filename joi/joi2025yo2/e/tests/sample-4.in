7 100 100
34 12 46 23 57 63 99
12 34 23 12 34 12 23
5
67 34
99 23
33 34
99 12
23 12
