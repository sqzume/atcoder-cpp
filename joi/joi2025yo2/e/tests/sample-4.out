330
264
341
440
341
